lib/core/presentation/pages/event/event_detail_page/guest_event_detail_page/sub_pages/guest_event_reward_uses_page/widgets/guest_event_reward_uses_listing.dart

```diff
@@ -69,9 +68,9 @@ class GuestEventRewardUsesListing extends StatelessWidget {
                     );
                   },
                 ),
-                HorizontalRewardsList(
-                  reward: reward,
-                ),
+                // HorizontalRewardsList(
```

Comment:
should we add TODO ?

Reply:
my bad, this should open instead of comment out